_source/_posts/2022-02-25-thymeleaf-security.md

```diff
@@ -0,0 +1,670 @@
+---
+layout: blog_post
+title: "Securing your application when using Thymeleaf templates with Spring Webflux"
```

Comment:
This title is a little long. I'll try to think of a better one. We also might want to add `spring` and/or `webflux` to the filename.

Reply:
maybe "Thymeleaf Templates with Spring WebFlux to Secure Your Apps" or similar?

Reply:
Works for me!

Reply:
Updated